### PR TITLE
A configuration for creating hrmpChannels between the parachains

### PR DESCRIPTION
**Pull request by pepyakin**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -137,6 +137,17 @@ async function main() {
 			await setBalance(relayChainApi, account, balance)
 		}
 	}
+
+	// wait here until the next session. Currently it's configured to be 5 blocks
+	// with 6 seconds per block. We wait for 2 sessions just in case.
+	let secsToSleep = 2 * 5 * 6
```
Comment (bkchr) [inline src/index.js:143]: Hard coding this is not a good idea :see_no_evil: 

There exists a constant `EpochDuration` in babe that you can read to get the number of blocks per session.

File: src/rpc.js [hunk 2]
```diff
@@ -105,3 +105,63 @@ export async function setBalance(api, who, value) {
...
+	nonce += 1;
+}
+
+export async function sendHrmpMessage(api, recipient, data) {
+	await cryptoWaitReady();
+
+	const keyring = new Keyring({ type: 'sr25519' });
+	const alice = keyring.addFromUri('//Alice');
+
+	if (!nonce) {
```
Comment (bkchr) [inline src/rpc.js:142]: Where is this nonce defined? :see_no_evil:

Comment (pepyakin) [reply src/rpc.js:142]: it's a global variable defined at the top of the file 🤫

File: src/index.js (hunk 1 above)
Comment (pepyakin) [reply src/index.js:143]: Alternatively, we can configure it ourselves. Ideally, though, we just wait for the right condition, specifically, that the both parachains are registered.

Comment (pepyakin) [reply src/index.js]: Done just that in the d9b6708e5175244aa5c83685aa5d0633b837a1fe

File: src/rpc.js (hunk 2 above)
Comment (pepyakin) [reply src/rpc.js:142]: Filed an issue for refactoring this
https://github.com/paritytech/polkadot-launch/issues/46

**Pull request merged**